### PR TITLE
icnlsp-ingestion

### PR DESCRIPTION
**Pull request by xinru1414**

### CODE REVIEW

File: data/xml/2021.icnlsp.xml [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+      <month>12--13 November</month>
+      <year>2021</year>
+      <url hash="b07b58ab">2021.icnlsp-1</url>
+    </meta>
+    <frontmatter>
+      <url hash="ab970842">2021.icnlsp-1.0</url>
+      <bibkey>icnlsp-2021-international</bibkey>
+    </frontmatter>
+    <paper id="1">
+      <title>Gerhard Hagerer, <fixed-case>D</fixed-case>avid Szabo, Andreas <fixed-case>K</fixed-case>och, Maria Luisa Ripoll Dominguez, Christian Widmer, Maximilian Wich, Hannah Danner and Georg Groh. End-to-End Annotator Bias Approximation on Crowdsourced Single-Label Sentiment Analysis</title>
```
Comment (mjpost) [inline data/xml/2021.icnlsp.xml:19]: This doesn't look right.

File: data/xml/2021.icnlsp.xml [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+      <author><first>Christian</first><last>Widmer</last></author>
+      <author><first>Maximilian</first><last>Wich</last></author>
+      <author><first>Hannah</first><last>Danner</last></author>
+      <author><first>Georg</first><last>Groh</last></author>
+      <pages>1–10</pages>
+      <url hash="0b94d867">2021.icnlsp-1.1</url>
+      <bibkey>hagerer-etal-2021-gerhard</bibkey>
+    </paper>
+    <paper id="2">
+      <title>Toshiko Shibano, Xinyi <fixed-case>Z</fixed-case>hang, Mia Taige <fixed-case>L</fixed-case>i, Haejin Cho, Peter <fixed-case>S</fixed-case>ullivan and Muhammad Abdul-Mageed. Speech Technology for Everyone: Automatic Speech Recognition for Non-Native <fixed-case>E</fixed-case>nglish</title>
```
Comment (mjpost) [inline data/xml/2021.icnlsp.xml:33]: Same here.

**Review by mjpost — COMMENTED** [review]
It looks like all author names were ingested as part of the paper titles.

**Comment by xinru1414** [comment]
@mjpost thanks for catching it. It turned out that the bib files were wrong. I fixed them.

**Review by mjpost — APPROVED** [review]
I verified a few PDFs. LGTM.

**Pull request merged**